src/sql/workbench/services/objectExplorer/browser/dragAndDropController.ts

```diff
@@ -45,6 +46,13 @@ export class ServerTreeDragAndDrop implements IDragAndDrop {
 		}
 	}
 
+	public static checkNodeTypeSingle(nodeId: string): boolean {
```

Comment:
1. Would rather we give this a more descriptive name of what it's actually doing - something like supportsNodeNameDrop
2. And then in addition it would be good to make a similar function for the folder support - supportsNodeNameDropForChildren possibly
3. These don't really need to be a part of the class - would suggest just having it be a module level function